Modules/LearningSequence/classes/GlobalScreen/class.ilLSToastProvider.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+use ILIAS\GlobalScreen\Identification\IdentificationInterface;
+use ILIAS\GlobalScreen\Scope\Toast\Provider\AbstractToastProvider;
+use ILIAS\Notifications\ilNotificationOSDHandler;
+use ILIAS\Notifications\Repository\ilNotificationOSDRepository;
+
+class ilLSToastProvider extends AbstractToastProvider
+{
+    private const NOTIFICATION_TYPE = ilLSCompletionNotificationProvider::NOTIFICATION_TYPE;
```

Comment:
Even if current Toast providers still have a dependency on Notification the goal is to seperate those 2 completly in future, therefore i would not recommend to build such strong dependency on notification here.

Neverless, since that is still a thing inside the core its more a suggestion here and no mandatory change request.

Reply:
OK, that's valid; however, the main goal here was the toast and it certainly depends on this exact notification-type.
I'd rather keept it for now and split it up when there are more notes or different toasts.